pkg/model/crd.go

```diff
@@ -494,10 +494,21 @@ func (r *CRD) GoCodeSetInput(
 		out += fmt.Sprintf("%s%s.SetAttributes(attrMap)\n", indent, targetVarName)
 	}
 
+	opConfig, override := r.genCfg.OverrideValues(op.Name)
+
 	for memberIndex, memberName := range inputShape.MemberNames() {
 		if r.UnpacksAttributesMap() && memberName == "Attributes" {
 			continue
 		}
+
+		if override {
+			value, ok := opConfig[memberName]
+			if ok {
+				out += fmt.Sprintf("%s%s.Set%s(%s)\n", indent, targetVarName, memberName, value)
```

Comment:
Hmm, this works for `SetSetImmediately(true)` only because the underlying data type is boolean... But this will blow up for any string type, which would expect `SetField("value")`.

We are going to need to a type switch here that inspects the Shape.Type for the InputShape field in question here and outputs Go code that will handle string types properly.

Reply:
Agreed, Added a new switch statement to handle `string`, `integer`, `boolean`.